test/unit/controller-cert-manager-serviceaccount.bats

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bats
+
+load _helpers
+
+@test "controller-cert-manager/ServiceAccount: disabled by default" {
+  cd `chart_dir`
+  assert_empty helm template \
+      -s templates/controller-cert-manager-serviceaccount.yaml  \
+      .
+}
+
+@test "controller-cert-manager/ServiceAccount: enabled with controller.enabled=true" {
+  cd `chart_dir`
+  local actual=$(helm template \
+      -s templates/controller-cert-manager-serviceaccount.yaml  \
+      --set 'controller.enabled=true' \
+      . | tee /dev/stderr |
+      yq 'length > 0' | tee /dev/stderr)
+  [ "${actual}" = "true" ]
+}
```

Comment:
Needs test for imagePullSecrets. See other serviceaccount.bats tests for details. If you were copying off another test that didn't have it that one should be updated too.